src/Constants/keys.py

```diff
@@ -0,0 +1,2 @@
+FREEZER = "freezer"
+FRIDGE = "fridge"
```

Comment:
I thought about putting in the same file as `commands_messages.py` but they weren't messages. So I thought it'd be better to make another constants file of keys (since these are keys for the `foodStorages` dictionary in the `Profile` class). We could potentially also add the commands that would be listed in `help()` here too.